.travis.yml

```diff
@@ -49,6 +51,8 @@ matrix:
     - compiler: clang
       env: PRESET=CC3200 XI_MAKETARGET=""
```

Comment:
yes, I can accept this, my reason was no keep simple the tutorial: make PRESET=CC3200 instead of make PRESET=CC3200_SDK110
The first visitor is undermotivated to get insights. But if you consider this and say, indeed we need CC3200_SDK_110 I'll do it.
Or just make an alias? CC3200->CC3200_SDK110?